nuxeo-java-client/src/main/java/org/nuxeo/client/api/objects/Document.java

```diff
@@ -20,6 +20,7 @@
 
 import java.lang.reflect.Field;
 import java.lang.reflect.Modifier;
+import java.time.temporal.Temporal;
```

Comment:
I though about it. However, when I see the [compiler configuration in POM](https://github.com/nuxeo/nuxeo-java-client/blob/3ed1cdc82886549690bbcbdbfdedf3d62110d974/pom.xml#L422-L430), it seems that we're using Java 8 instead of Java 7, that's why I still added the support for `java.time`. Should I change the compiler option back to Java 7?

Otherwise, I agree with you, I'll remove this line.

As for test, I'd like to keep them. They're not only test cases, but also like part of the demo.